### ISSUE TITLE
wo update returns exit code 1

### ISSUE THREAD

**Issue by tersor** [WordOps/WordOps#193]
Hello!

If latest wordops version is already installed, a `wo update` returns exit code 1:

```
wo update
WordOps v3.10.2 is already installed

echo $?
1
```

Since this is more an info message rather than an error, I suggest returning 0 (zero) by using the `Log.info` method [here](https://github.com/WordOps/WordOps/blob/master/wo/cli/plugins/update.py#L68).

**Comment by vikas5914** [comment]
Created PR #194

**Comment by VirtuBox** [comment]
Hello @tersor,
yes this is an info message, but without using Log.error, WordOps will still try to update itself even if the latest release is already installed.
To force update there is the flag `--force` available.

**Comment by tersor** [comment]
@VirtuBox,
Oh, I see. Perhaps the `Log.info` method can be adjusted so it is possible to terminate further execution with exit code 0? And only if `--force` is not present...

```
class Log:
...
def info(self, msg, end='\n', log=True, exit=False):
...
    if exit:
        self.app.close()
```
(I assume `self.app.close()` exits with default code 0).

Or alternatively, adjust the `def error` method so it has the possibility to exit with code 0.

**Comment by vikas5914** [comment]
OR

on update.py#L68 , you can just do `exit(0)` which is a python inbuilt function, which will stop the execution of the function.

**Comment by VirtuBox** [comment]
Fixed with commit 642605838adbab9a83eb4112a0f447977ebf4888.

**Comment by VirtuBox** [comment]
Issue has been fixed in the last release.

Please update WordOps with the command `wo update`
